README.md

```diff
@@ -20,31 +20,76 @@ Emacs users may install the package from MELPA, and then add the following to th
 
 ## Spacemacs
 
-Add
-```lisp
-(vue-mode :location (recipe
-                     :fetcher github
-                     :repo "codefalling/vue-mode"))
-```
-
-to your `packages`.and
-
-```lisp
-(defun your-layer-name/init-vue-mode ()
-  (use-package vue-mode))
-```
-
-to your `package.el`.
-
-And you could custom the region background color if you need (default is highlight):
-
-```lisp
-(defun your-layer-name/init-vue-mode ()
-  (use-package vue-mode
-    :config
-    ;; 0, 1, or 2, representing (respectively) none, low, and high coloring
-    (setq mmm-submode-decoration-level 0)))
-```
+### There are [two ways](http://spacemacs.org/doc/DOCUMENTATION.html#configure-packages) to install and configure Spacemacs Packages:
+
+#### A. The fastest way: Adding the necessary configurations to the default configuration file `.spacemacs`
+
+   1. Within any screen in Spacemacs (including the Splash Screen, after loading), press `SPC` `f` `e` `d`.
+   2. This will bring you to the edit screen of the `.spacemacs` file.
+   3. Locate the section named `dotspacemacs-additional-packages '()`
+   4. The following code should go inside the above parenthesis:
+   ```lisp
+   (vue-mode :location (recipe
+                        :fetcher github
+                        :repo "codefalling/vue-mode"))
+   ```
+   • So the section should look like this, after the update:
+   ```lisp
+   dotspacemacs-additional-packages '(
+   (vue-mode :location (recipe
+                        :fetcher github
+                        :repo "codefalling/vue-mode"))
+   )
+   ```
+   5. Locate, in the same file, the section named `defun dotspacemacs/user-config ()`
+   6. The following code should code right in that section:
+   ```lisp
+   (defun dotspacemacs/init-vue-mode ()
+     (use-package vue-mode))
+   )
+   ```
+   • So the section should look like this, after the update:
+   ```lisp
+   (defun dotspacemacs/user-config ()
+   "Configuration function for user code.
+   This function is called at the very end of Spacemacs initialization after
+   layers configuration.
+   This is the place where most of your configurations should be done. Unless it is
+   explicitly specified that a variable should be set before a package is loaded,
+   you should place your code here."
+   (defun dotspacemacs/init-vue-mode ()
+     (use-package vue-mode))
+   )
+   )
+   ```
+   7. So we made 2 changes to the `.spacemacs` file, time to save them, press `SPC` `f` `s`
+   8. With the saved file, time to make Spacemacs re-read the configuration, pressing `SPC` `f` `e` `R`
+
+#### B. The [more complex](http://spacemacs.org/doc/DOCUMENTATION.html#without-a-layer) way: Creating a new specific layer 
+
+   1. Add to your `packages`:
+   ```lisp
+   (vue-mode :location (recipe
+                        :fetcher github
+                        :repo "codefalling/vue-mode"))
+   ```
+   2. Add to your `package.el`:
+
+   ```lisp
+   (defun your-layer-name/init-vue-mode ()
```

Comment:
@jaysoifer who is calling this function? Is this a way to have Spacemacs call "init" function, like calling them "init-$mode-name"?